service/src/main/java/bio/terra/axonserver/app/controller/AuthorizationApiController.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.axonserver.app.controller;
+
+import bio.terra.axonserver.api.AuthorizationApi;
+import bio.terra.axonserver.model.ApiTokenReport;
+import bio.terra.common.exception.ApiException;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeTokenRequest;
+import com.google.api.client.googleapis.auth.oauth2.GoogleRefreshTokenRequest;
+import com.google.api.client.googleapis.auth.oauth2.GoogleTokenResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class AuthorizationApiController implements AuthorizationApi {
+  private final HttpServletRequest servletRequest;
+
+  @Value("${AXON_UI_CLIENT_ID}")
```

Comment:
i think this is a fine pattern given this is the only place using these env vars. If we ever need to use them in multiple places, we should adapt the wsm pattern where we add these two fields to application.yaml profile and set them to these two environment variables.